### DISCUSSION TITLE
Translation in Hindi

### DISCUSSION THREAD

**Discussion by mohakdev** [DevToys-app/DevToys#513] [Ideas]
I believe we should translate the app in Hindi language which is the 3rd Most spoken language in the world.
I would love to translate the app in Hindi language so the app becomes accessible to more people.

**Comment by veler** [comment]
Hello,
Thanks for sharing this feedback. I don't speak Hindi personally, but please feel free to provide a translation by following the guidance here: https://github.com/veler/DevToys/blob/main/CONTRIBUTING.md#internationalization-and-localization

**Reply by mohakdev** [reply]
Understood. I would be happy to contribute
